# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/scheduler/tentative/yield/yield-priority-timers.any.js.ini
# Commit: 4ccb6970887e
# Full Hash: 4ccb6970887ece212a1176681a567480ea586dfc
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2025-02-25 09:30:56
# Regressor Bug: 1920115
# File Overlap Count: 2
# Description:
#   Bug 1920115 - Allow high priority tasks to run before timers r=smaug
#   
#   In Gecko, unlike other runnables where the TaskController
#   picks one each at a time, the TimeoutManager allows multiple
#   timers to run sequentially without allowing other runnables
# ==============================================================================

diff -r 7522970c5a1d -r 4ccb6970887e testing/web-platform/meta/scheduler/tentative/yield/yield-priority-timers.any.js.ini
--- a/testing/web-platform/meta/scheduler/tentative/yield/yield-priority-timers.any.js.ini	Mon Feb 24 19:31:28 2025 +0000
+++ b/testing/web-platform/meta/scheduler/tentative/yield/yield-priority-timers.any.js.ini	Mon Feb 24 19:31:28 2025 +0000
@@ -1,8 +1,6 @@
 [yield-priority-timers.any.worker.html]
+  # Currently worker doesn't use the TimeoutManager.
+  # This test needs to be investigated when `dom.workers.timeoutmanager` is enabled.
+  # Bug 1947484
   [yield() with timer tasks (inherit signal)]
     expected: FAIL
-
-
-[yield-priority-timers.any.html]
-  [yield() with timer tasks (inherit signal)]
-    expected: FAIL
